api_tests/utils.py

```diff
@@ -15,3 +15,30 @@ def create_test_file(node, user):
         'contentType': 'img/png'
     }).save()
     return test_file
+
+
+def create_test_file_for_size_filter(node, user):
```

Comment:
This is super handy, though we have factories for most things (UserFactory, ProjectFactory, etc). Look at making a factory like the others rather than doing it this way.